packages/playwright-core/src/server/transport.ts

```diff
@@ -24,6 +24,8 @@ import { httpHappyEyeballsAgent, httpsHappyEyeballsAgent } from '../utils/happy-
 import type { HeadersArray } from './types';
 
 export const perMessageDeflate = {
+  serverNoContextTakeover: true,
```

Comment:
I believe (based on [this part](https://github.com/websockets/ws/blob/68614728c164eb55462ff12ba24cb30451d28ec6/lib/sender.js#L372-L375) of the implementation) that `serverNoContextTakeover` doesn't have any effect on the client web socket. If so let's not set it in the options here.

Reply:
Good catch, thanks.